lib/currency-for.js

```diff
@@ -0,0 +1,9 @@
+var currencies = require('../data/currencies.js');
+module.exports = {
+  stripe: Object.keys(currencies),
+  paypal: Object.keys(currencies).filter(function(i) {
+    if (!currencies[i].hasOwnProperty('disabled')) {
```

Comment:
Stripe basically doesn't have that property at all and I believe we want all currencies in the list, so not sure if we need this now.